apps/user/components/tips/index.coffee

```diff
@@ -7,6 +7,7 @@ module.exports = ->
   tips = new Backbone.Collection TIPS
 
   tips.each (tip) ->
+    return if Cookies.get(tip.id) 
     new TipView
       el: $(".js-tip[data-id='#{tip.id}']")
       model: tip
```

Comment:
This one doesn't make sense to me. All this is doing is binding (or not) a view.